LICENSE.md

```diff
@@ -565,7 +565,6 @@ Ruffle depends on third-party libraries with compatible licenses.
 | [time](https://github.com/time-rs/time) | [Apache-2.0](#Apache-20)/[MIT](#MIT) | Copyright (c) 2019 Jacob Pratt  | 
 | [time-macros](https://github.com/time-rs/time) | [Apache-2.0](#Apache-20)/[MIT](#MIT) | Copyright (c) 2019 Jacob Pratt  | 
 | [time-macros-impl](https://github.com/time-rs/time) | [Apache-2.0](#Apache-20)/[MIT](#MIT) | Copyright (c) 2019 Jacob Pratt  | 
-| [tinyfiledialogs](https://github.com/jdm/tinyfiledialogs-rs) | zlib-acknowledgement | Copyright (c) Guillaume Vareille <vareille@users.sourceforge.net>, Josh Matthews <josh@joshmatthews.net> | 
```

Comment:
Scripting would be better. I thought this was just a list of those that required special acknowledgement, but it does seem to be the bulk of those present.

Reply:
The list was actually auto-generated with a script I wrote, with some manual tweaks here and there -- I'll try to polish the script up so that this can be more automated.

Reply:
Can't we use `cargo licence` for this? Does this need to be embedded in the repo at all?